server/service/team_schedule.go

```diff
@@ -0,0 +1,253 @@
+package service
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/fleetdm/fleet/v4/server/ptr"
+	kithttp "github.com/go-kit/kit/transport/http"
+	"gopkg.in/guregu/null.v3"
+)
+
+type getTeamScheduleRequest struct {
+	TeamID      uint              `url:"team_id"`
+	ListOptions fleet.ListOptions `url:"list_options"`
```

Comment:
Here's how you use the decoders. Maybe we want to rename the tag `fleet` to potentially avoid collisions, not sure.